### PR TITLE
Enhancement: Build our Gherkin as a sample.

### PR DESCRIPTION
**Pull request by dgou**
This is pretty rough, so I'm expecting wordsmithing and other feedback.

### CODE REVIEW

**Comment by dgou** [comment]
Going to wait for jgt_tools before updating self-check to build the docs.

**Comment by dgou** [comment]
If you pull this PR and run `bin/build_docs.py` you can open `docs/index.html` and browse to the Ghkerin and Glossary.
Travis builds the docs, but I wasn't able to figure out if they were left behind anywhere like Jenkins does.

File: bin/build_docs.py [hunk 1]
```diff
@@ -62,6 +62,23 @@ def main():
...
+            "--doc-project",
+            "Sample Gherkin",
+            "--toc-name",
+            "sample-gherkindoc",
+            "--step-glossary-name",
+            "sample-gherkindoc-glossary",
+            "tests",
+            DOCS_WORKING_DIRECTORY,
+        ]
+    )
```
Comment (rbcasperson) [inline bin/build_docs.py]: I think that for #11 at least, I'd end up making another example like this to show what it looks like with the background integrated.

Comment (dgou) [reply bin/build_docs.py]: I think that would be a good idea. I'm a big fan of "Ok, yeah, I read how this works, but _show me_ what it looks like." So if we have multiple options like that, we should definitely show them off!

**Pull request merged**